components/message/style/index.less

```diff
@@ -29,6 +29,10 @@
     pointer-events: all;
   }
 
+  &-notice-content-text {
+    word-break: break-word;
```

Comment:
It will break English words which may not be expected for every one. I believe you should add this styles to your project if you need this.